dask_saturn/core.py

```diff
@@ -2,19 +2,41 @@
 import time
 import requests
 import json
+
+from datetime import datetime
+from random import randrange
 from urllib.parse import urljoin
 from distributed import SpecCluster
+from typing import List, Dict
+from math import ceil
 
 
 SATURN_TOKEN = os.environ.get("SATURN_TOKEN", "")
 BASE_URL = os.environ.get("BASE_URL", "")
 HEADERS = {"Authorization": f"token {SATURN_TOKEN}"}
+DEFAULT_WAIT_TIMEOUT_SECONDS=1200
 
 
 class SaturnCluster(SpecCluster):
-    def __init__(self, cluster_url=None, *args, **kwargs):
+    def __init__(
+        self,
+        cluster_url=None,
+        worker_size=None,
+        scheduler_size=None,
+        nprocs=None,
+        nthreads=None,
+        scheduler_service_wait_timeout=DEFAULT_WAIT_TIMEOUT_SECONDS,
+        *args,
```

Comment:
The args needs to go before the defined kwargs.

```suggestion
        self,
        *args,
        cluster_url=None,
        worker_size=None,
        scheduler_size=None,
        nprocs=None,
        nthreads=None,
        scheduler_service_wait_timeout=DEFAULT_WAIT_TIMEOUT_SECONDS,
```